pkg/gopass/secrets/secparse/parse.go

```diff
@@ -25,14 +27,16 @@ func Parse(in []byte) (gopass.Secret, error) {
 		debug.Log("parsed as YAML: %+v", s)
```

Comment:
Here too actually, if there is no error, we log the secret, don't we?

Reply:
No, `secrets.YAML` implements the `Safer` interface, thus it's being rewritting inside the debug package: https://github.com/gopasspw/gopass/pull/1886/files#diff-2769ebba150a804d5515d28f28b7a8e06f05d914e9781b4e58beac1dc4d2ed52R203